spec/integration_spec.rb

```diff
@@ -228,6 +228,7 @@ def test_unicode
   end
 
   it "runs with PARALLEL_TEST_PROCESSORS processes" do
+    skip "fails on jruby" if RUBY_PLATFORM == "java"
```

Comment:
this worked on jruby in this build: https://travis-ci.org/grosser/parallel_tests/builds/71348072
maybe just a random failure ?